server/src/main/java/org/opensearch/plugins/ClusterPlugin.java

```diff
@@ -87,6 +88,16 @@ default Map<String, ExistingShardsAllocator> getExistingShardsAllocators() {
     /**
      * Called when the node is started
      */
+    @Deprecated
```

Comment:
Nit: could you include instructions to use the other `onNodeStarted(DiscoveryNode)` instead?